tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,410 @@
+            "to_address": "cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad", // a third-party
+            "amount": [
+                {
+                    "denom": "stake",
+                    "amount": "100"
+                }
+            ]
+        }
+    ],
+    "metadata": "utilities bill",
+    "proposers": [ "bobaddr" ] // $BOB
+}
+```
+
+This proposal, if passed, will send 100 `stake` to `cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad` to pay the bills.
+The tokens will be sent from the decision policy address.
+
+<HighlightBox type="tip">
+
+The decision policy has no funds yet. You can fund it by sending a transaction with `simd tx bank send alice $GROUP_POLICY_ADDRESS 100stake`.
+
+</HighlightBox>
+
+Submit the proposal:
+
+```sh
+$ simd tx group submit-proposal proposal.json --from bob
+```
+
+Once more, extract the proposal ID (remember to use the transaction hash you got at the previous command):
+
+```sh
+$ export PROPOSAL_ID=$(simd query tx E3CBE6932254088D5A80CD5CB18BB0F4D35396A542BD20731E1B6B997E1B0847 --output json | jq '.events' | jq -r '.[] | select(.type == "cosmos.group.v1.EventSubmitProposal") | .attributes[0].value' | base64 --decode | jq -r '.'
+```
+
+## View and vote on proposals
+
+You can see that your proposal has been submitted. And that it contains a lot of information. For instance, confirm that its final tally is empty:
+
+```sh
+$ simd query group proposals-by-group-policy $GROUP_POLICY_ADDRESS --output json | jq '.proposals[0].final_tally_result'
+```
+
+Which returns:
+
+```json
+{
+  "yes_count": "0",
+  "abstain_count": "0",
+  "no_count": "0",
+  "no_with_veto_count": "0"
+}
+```
+
+And that it is in the `PROPOSAL_STATUS_SUBMITTED` status:
+
+```sh
+$ simd query group proposals-by-group-policy $GROUP_POLICY_ADDRESS --output json | jq -r '.proposals[0].status'
+```
+
+Next, have Alice and Bob vote _Yes_ on the proposal and verify that both their votes are tallied using the proper query command:
+
+```sh
+$ simd tx group vote $PROPOSAL_ID $ALICE VOTE_OPTION_YES "agree"
+$ simd tx group vote $PROPOSAL_ID $BOB VOTE_OPTION_YES "aye"
+$ simd query group tally-result $PROPOSAL_ID
+```
+
+While you wait for the policy-prescribed 10 minutes, you can confirm that the final tally is still empty. After the 10 minutes have gone by your proposal should have passed, because the weighted tally of _Yes_ votes is above the decision policy threshold. Confirm this by looking at its `status`. It should be `PROPOSAL_STATUS_ACCEPTED`:
+
+```sh
+$ simd query group proposal $PROPOSAL_ID
+```
+
+By default proposals are not executed immediately. You can confirm this by looking at the proposal, it contains `executor_result: PROPOSAL_EXECUTOR_RESULT_NOT_RUN`. This is to account for the fact that not everything may be in place to successfully execute the proposal's messages. As you recall, you already funded the group policy. If you did not fund it ahead of time, now is the time to do it
```

Comment:
```suggestion
By default proposals are not executed immediately. You can confirm this by looking at the proposal, it contains `executor_result: PROPOSAL_EXECUTOR_RESULT_NOT_RUN`.

<HighlightBox type="remember">

This is to account for the fact that not everything may be in place to successfully execute the proposal's messages. As you recall, you already funded the group policy. If you did not fund it ahead of time, now is the time to do it. 

Another advantage of execution by way of a transaction is that it shifts the cost of the messages onto a network participant. If the cost was born in an end blocker, it would open the door to gas arbitrage and outright spam attacks.

</HighlightBox>
```

Reply:
Partially added

Reply:
Do you think the end blocker remark does not apply?
Or too far a remark from the goal of the tutorial?

Reply:
It's a valid remark, however none of the CLI tutorials discuss the internals, I thought it'd be fine to leave that out.